### PR TITLE
Added wordpress ansibleapp

### PR DESCRIPTION
**Pull request by dymurray**

### CODE REVIEW

File: wordpress-ansibleapp/README.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+Role wordpress created.
+
+```
+It will create the OpenShift artifacts to deploy it on OpenShift.
+
+
+## Run the OpenShift artifacts.
+
+```
+$ oc create -f wordpress/ansible/shipit_config/openshift/
```
Comment (fabianvf) [inline wordpress-ansibleapp/README.md]: Out of curiosity, why run the oc artifacts instead of the playbook?

Comment (fabianvf) [reply wordpress-ansibleapp/README.md]: Also, when we add variable support (ie, configurable number of replicas etc), will they work with this method?

Comment (dymurray) [reply wordpress-ansibleapp/README.md]: This is just from the README I took from the ansible-container-examples. I will remove all this.

**Comment by fabianvf** [comment]
ACK

**Comment by detiber** [comment]
Only tangentially related to this PR, is there any effort being made to make Wordpress scale-able without the need to use ReadWriteMany volumes with this effort?

**Comment by dymurray** [comment]
@detiber I am happy to look into this as we further flesh out this library. This is just meant to be some easy simple to consume apps for a first cut.

**Pull request merged**